Fixed include and dependency

Xottab-DUTY opened the pull request:
Fixed include (wrong path to xrCore in xrD3D9-Null) and dependency (You need build xrLCUtil before building xrLC_Light)

Kaffeine commented on src/xrD3D9-Null/xrD3D9-Null.h:
```diff
@@ -3,7 +3,7 @@
 #include "IDirect3D9.h"
 
-#include "xrCore/Platform.h"
+#include "../xrCore/Platform.h"
```
Don't we have "src" in include path list?
I think we should use "module/path_in_the_module/header_file" scheme instead of relative paths.
I'd like to see unified includes instead of things like "../../../xrCore/something.h", which're really error-prone and would break on move.

nitrocaster replied:
xray-16/src is one of default include search paths.
upd: It should be, but xrD3D9-Null doesn't define it. Project settings need to be adjusted.

Xottab-DUTY committed:
Fixed include and dependency

Fixed include (missing path to xrCore in xrD3D9-Null) and dependency
(You need build xrLCUtil before building xrLC_Light)
